### DISCUSSION TITLE
How to keep color/translucency of taskbar after applying a theme?

### DISCUSSION THREAD

**Discussion by laurorual** [ramensoftware/windhawk#257] [Q&A]
So, i'm trying to apply this theme on Taskbar Styler: https://www.vinstartheme.com/windows-11x-taskbar/
But, no matter what i do my taskbar goes black and looses the translucency, something that shouldn't happen (given how in the screenshots and the video i've seen the color/translucency of the taskbar keeps the same as before).
How do i fix this?

How it should be:
![Captura de tela 2024-08-23 165751](https://github.com/user-attachments/assets/175350cf-f3ac-44cf-8e66-5c06528e55d7)

How it is now:
![Captura de tela 2024-08-23 165811](https://github.com/user-attachments/assets/c833ecba-6344-495e-925b-bb2d6c52d454)

**Comment by m417z** [comment]
Here would be a better place to ask this: https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues
Also, have you tried asking the theme authors?

**Reply by laurorual** [reply]
Thanks! I just posted an issue there as well.
As for asking the author, i don't think it's this theme's fault, cuz the same happens with other themes that should follow color/translucency but it's not.
